Framework/PythonInterface/mantid/api/CMakeLists.txt

```diff
@@ -143,17 +143,13 @@ if(CMAKE_GENERATOR STREQUAL Xcode)
   set ( CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PYTHON_PKG_ROOT}/api )
 endif()
 
-copy_files_to_dir ( "${PY_FILES}" ${CMAKE_CURRENT_SOURCE_DIR} ${OUTPUT_DIR}
```

Comment:
@martyngigg I believe these changes fix the issue of not having `_api.pyd` in `bin/` for dev builds on Linux. It should work now

Reply:
:+1: